tools/bigquery-to-xml/bq_to_xml.py

```diff
@@ -81,7 +81,7 @@ def bigquery_to_xml(query, custom_root_node="results", custom_row_tag="row"):
 
 
     # Convert to string
-    my_xml = ET.tostring(doc, method="xml")
+    my_xml = ElementTree.tostring(doc, method="xml")
```

Comment:
```suggestion
    my_xml = xml.etree.ElementTree.tostring(doc, method="xml")
```

Reply:
Done - I should have caught this before, but it looks like I can use lxml for everything to avoid the confusion of it all.